src/main.py

```
@@ -7,49 +7,96 @@
+    print("2 - Sepia")
+    print("3 - Invert")
+    print("4 - Convolution (Sharpening)")
+    print("5 - Histogram Equalization")
+    print("6 - Edge Detection")
 
-    parser = argparse.ArgumentParser(description='Process images with filters and math utils.')
-    parser.add_argument('--input', default=os.path.join("data", "input", "croco.jpeg"), help='Path to input image')
-    parser.add_argument('--output', default=os.path.join("data", "output", "croco.jpg"), help='Path to output image')
-    args = parser.parse_args()
+def apply_selected_filters(image, filters, output_dir, logger):
+    for i, f in enumerate(filters, start=1):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)